src/main/java/com/github/luben/zstd/Zstd.java

```diff
@@ -772,7 +773,7 @@ public static long trainFromBufferDirect(ByteBuffer samples, int[] sampleSizes,
      * @return byte array with the compressed data
      */
     public static byte[] compress(byte[] src) throws ZstdException {
-        return compress(src, 3);
+        return compress(src, defaultCompressionLevel());
```

Comment:
I don't know much about JNI, but I can imagine that this change comes with a (super tiny) performance hit. An alternative would be to store the result of `defaultCompressionLevel()` in a `private static final int` constant and reuse it here and below.